src/renderer/components/ft-community-post/ft-community-post.vue

```diff
@@ -119,11 +119,26 @@
         class="thumbs-up-icon"
         :icon="['fas', 'thumbs-up']"
       /> {{ voteCount }}</span>
-      <span class="commentCount">
-        <font-awesome-icon
-          class="comment-count-icon"
-          :icon="['fas', 'comment']"
-        /> {{ commentCount }}</span>
+      <router-link
+        v-if="isInvidiousAllowed && !singlePost"
+        :to="`/post/${postId}`"
+        class="commentsLink"
+      >
```

Comment:
Might be worth adding a title that says something along the lines of `View Comments` or `View Full Post`, as the link body by itself (icon + number) will be confusing for people using screen readers.